android/anc/src/test/java/org/smartregister/fhircore/anc/data/report/ReportRepositoryTest.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.anc.data.report
+
+import androidx.paging.PagingConfig
+import androidx.paging.PagingState
+import com.google.android.fhir.FhirEngine
+import io.mockk.mockk
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Test
+import org.smartregister.fhircore.anc.data.report.model.ReportItem
+import org.smartregister.fhircore.anc.robolectric.RobolectricTest
+
+@ExperimentalCoroutinesApi
+class ReportRepositoryTest : RobolectricTest() {
+
+  private lateinit var repository: ReportRepository
+  private lateinit var fhirEngine: FhirEngine
+
+  @Before
+  fun setUp() {
+    fhirEngine = mockk()
+    repository = ReportRepository(fhirEngine, "")
+  }
+
+  @Test
+  fun testGetRefreshKeyShouldReturnValidAnchorPosition() {
+    val pagingState = PagingState<Int, ReportItem>(listOf(), 0, PagingConfig(1), 10)
+    val anchorPosition = repository.getRefreshKey(pagingState)
+    Assert.assertEquals(0, anchorPosition)
+  }
+
+  @Test
+  @Ignore("todo work")
```

Comment:
It would be okay if the reason why this test is ignored is more explicit.

Reply:
updated